Polyhedron/demo/Polyhedron/Viewer.cpp

```diff
@@ -15,6 +15,11 @@
 #include <cmath>
 #include <QApplication>
 
+#if defined(BOOST_MSVC)
```

Comment:
Instead of `BOOST_MSVC`, you should test something related to Windows. MSVC can target Android, and there are Windows compilers that are not MSVC.

I suggest `_WIN32`.